.github/workflows/ci.yml

```diff
@@ -154,7 +133,12 @@ jobs:
         if: github.ref == 'refs/heads/main'
         uses: changesets/action@v1
         with:
-          publish: npm run release
+          publish: |
+            npm run changeset-version
+            cd packages/python && uv publish --trusted-publishing always
```

Comment:
I don't think you're building the python package!

I think you need `uv build --package genai-prices` in there before publish?